infra/partner-mainnet/variables.tf

```diff
@@ -0,0 +1,134 @@
+variable "project_id" {
+  description = "The project ID to deploy resource into"
+  type        = string
+}
+
+variable "subnetwork" {
+  description = "The name of the subnetwork to deploy instances into"
+  type        = string
+}
+
+variable "mig_name" {
+  description = "The desired name to assign to the deployed managed instance group"
+  type        = string
+  default     = "mpc-mig"
+}
+
+variable "image" {
+  description = "The Docker image to deploy to GCE instances. Note: This is a public image repository used for updating your nodes, please do not change this"
+  type        = string
+  default     = "us-east1-docker.pkg.dev/pagoda-discovery-platform-prod/multichain-public/multichain-testnet:latest"
```

Comment:
this will need to be changed to `us-east1-docker.pkg.dev/pagoda-discovery-platform-prod/multichain-public/multichain-mainnet:latest`